scout/index.md

```diff
@@ -16,62 +16,60 @@ redirect_from:
 
 {% include scout-early-access.md %}
 
-Docker Scout is a collection of software supply chain features that appear
-throughout Docker user interfaces and the command line interface (CLI). These features provide detailed
-insights into the composition and security of container images.
+Container images are often built from layers of other container images and
```

Comment:
I think the current introductory section still makes sense as it clearly states what Scout is all about. Can we retain the intro section and then add this?

Reply:
Yes, this change was based on feedback from a few of the team.